apps/condo/pages/ticket/[id]/index.tsx

```diff
@@ -231,139 +254,160 @@ const TicketIdPage = () => {
                                             </Row>
                                         </Col>
-                                        <Row gutter={[0, 8]} style={{ maxWidth: '100%' }}>
-                                            <Col span={24}>
-                                                <Typography.Title level={5} style={{ margin: 0 }}>{TicketInfoMessage}</Typography.Title>
-                                            </Col>
-                                            <Col span={24}>
-                                                <Typography.Text style={{ fontSize: '24px', wordWrap: 'break-word' }}>{ticket.details}</Typography.Text>
-                                                {
-                                                    !isEmpty(files) && (
-                                                        <TicketFileList files={files} />
-                                                    )
-                                                }
-                                            </Col>
-                                        </Row>
-                                    </Row>
-                                </Col>
-                            </FocusContainer>
+                                    </FocusContainer>
+                                    <ActionBar>
+                                        <Link href={`/ticket/${ticket.id}/update`}>
+                                            <Button
+                                                color={'green'}
+                                                type={'sberPrimary'}
+                                                secondary
+                                                icon={<EditFilled />}
+                                            >
+                                                {UpdateMessage}
+                                            </Button>
+                                        </Link>
+                                        <Button
+                                            type={'sberPrimary'}
+                                            icon={<FilePdfFilled />}
+                                            href={`/ticket/${ticket.id}/pdf`}
+                                            target={'_blank'}
+                                            secondary
+                                        >
+                                            {PrintMessage}
+                                        </Button>
+                                    </ActionBar>
+                                    <TicketChanges
+                                        loading={ticketChangesResult.loading}
+                                        items={ticketChangesResult.objs}
+                                        total={ticketChangesResult.count}
+                                    />
+                                </Row>
+                            </Col>
+                            <Col span={1}>
```

Comment:
col without rows?

Reply:
`Col`, that You highlighted, does have parent `Row`